File: src/main/scala/com/mbpolan/ws/Application.scala
package com.mbpolan.ws

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.scheduling.annotation.EnableScheduling

/** Configuration for the web application.
  *
  * @author Mike Polan
  */
object Application extends App {
  SpringApplication.run(classOf[WebAppConfig], args: _*)
}

@SpringBootApplication
@EnableScheduling
class WebAppConfig

File: server/src/main/java/com/mbpolan/retrorealms/Application.java
package com.mbpolan.retrorealms;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

/**
 * Entry point into the application.
 *
 * @author mbpolan
 */
@SpringBootApplication
@EnableScheduling
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }
}
